db/primary_migrate/20221031203041_add_early_reminder_sent_and_late_reminder_sent_to_in_person_enrollments.rb

```diff
@@ -0,0 +1,6 @@
+class AddEarlyReminderSentAndLateReminderSentToInPersonEnrollments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :in_person_enrollments, :early_reminder_sent, :boolean, default: false
+    add_column :in_person_enrollments, :late_reminder_sent, :boolean, default: false
```

Comment:
please add comments describing the purpose of these columns; see [this migration](https://github.com/18F/identity-idp/blob/main/db/primary_migrate/20220808140030_add_issuer_to_in_person_enrollments.rb) as an example.